scipy/optimize/tests/test_slsqp.py

```diff
@@ -24,6 +26,44 @@ def __call__(self, x):
         self.ncalls += 1
 
 
+def _check_kkt(res, constraints):
```

Comment:
@Tenavi could you add the conditions for simple bounds to these checks? It looks like the relevant information would just need to be added into `mus` and `gs`.

Reply:
@mdhaber @andyfaff I made some updates to fix the bound multipliers, modified `_check_kkt` to test for these, and also added a specific test to make sure I was getting the indexing right. Please take a look at the PR when you can: [https://github.com/andyfaff/scipy/pull/42](https://github.com/andyfaff/scipy/pull/42)

I still would need to update the documentation for `minimize` as you suggested. I think your writeup looks good as is... I could push these changes as well, or were you looking for something a little more?

Reply:
You are welcome to use the example I suggested if it looks good to you.

Reply:
@mdhaber I added to the `minimize` documentation (thanks for your example) and made a test to verify the behavior.